packages/nimble-components/src/combobox/tests/combobox.spec.ts

```diff
@@ -1,259 +1,14 @@
-        >
-            ${repeat(() => [...Array(500).keys()], html<number>`
-                <nimble-list-option>${x => x}</nimble-list-option>
-            `)}
-            <nimble-list-option>1000</nimble-list-option>
-        </nimble-combobox>
-    `;
-    return fixture<Combobox>(viewTemplate);
-}
-
-function updateComboboxWithText(combobox: Combobox, text: string): void {
-    combobox.control.value = text;
-    const inputEvent = new InputEvent('input', {
-        data: text,
-        inputType: 'insertText'
-    });
-    combobox.inputHandler(inputEvent);
-    combobox.dispatchEvent(inputEvent);
-}
-
-async function clickAndWaitForOpen(combo: Combobox): Promise<void> {
-    const regionLoadedListener = createEventListener(combo, 'loaded');
-    combo.click();
-    await regionLoadedListener.promise;
-}
+import { ComboboxPageObject } from '../testing/combobox.pageobject';
 
 describe('Combobox', () => {
-    it('should set aria-disabled attribute when property is set', async () => {
-        const { element, connect, disconnect } = await setup();
-
-        element.ariaDisabled = 'true';
-        await connect();
-
-        expect(element.getAttribute('aria-disabled')).toBe('true');
-
-        await disconnect();
-    });
-
-    it('should set autocomplete attribute when property is set', async () => {
-        const { element, connect, disconnect } = await setup();
-
-        element.autocomplete = ComboboxAutocomplete.both;
-        await connect();
-
-        expect(element.getAttribute('autocomplete')).toBe(
-            ComboboxAutocomplete.both
-        );
-
-        await disconnect();
-    });
-
-    it('should set classes based on open, disabled, and position', async () => {
-        const { element, connect, disconnect } = await setup('above', true);
-
-        element.disabled = true;
-        await connect();
-        await waitForUpdatesAsync();
-
-        expect(element.classList.contains('above')).toBeTrue();
-        expect(element.classList.contains('disabled')).toBeTrue();
-        expect(element.classList.contains('open')).toBeTrue();
-
-        await disconnect();
-    });
-
-    it('should set tabindex based on disabled state', async () => {
-        const { element, connect, disconnect } = await setup();
-
-        await connect();
-
-        expect(element.getAttribute('tabindex')).toBe('0');
-        element.disabled = true;
-        await waitForUpdatesAsync();
-        expect(element.getAttribute('tabindex')).toBeNull();
-
-        await disconnect();
-    });
-
-    it('should forward  based on disabled state', async () => {
-        const { element, connect, disconnect } = await setup();
-
-        await connect();
-
-        expect(element.getAttribute('tabindex')).toBe('0');
-        element.disabled = true;
-        await waitForUpdatesAsync();
-        expect(element.getAttribute('tabindex')).toBeNull();
-
-        await disconnect();
-    });
```

Comment:
Removed `'should forward based on disabled state'` -- looks like a copy-paste leftover that is just a duplicate of the previous test.